server/.env

```diff
@@ -1,7 +1,10 @@
 # database config
 DB_USERNAME="postgres"
 DB_PASSWORD="postgres"
-CKB_NODE_URL="http://localhost:3333"
```

Comment:
It is not recommended to commit this file into the repo.

1. Usually, developers need to edit it locally.
2. It may expose sensitive information by accident.

The best practices are:

1. Provide an example file.
2. Add a script `bin/setup` to copy the example file to `server/.env` if it does not exist yet.
3. Add `server/.env` to `.gitignore`.

Reply:
thank you, this file just a template, not the real one. I will use the best practices you recommend.